apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js

```diff
@@ -119,13 +118,11 @@ describe('LessonEditor', () => {
         .at(2)
         .props().disabled
     ).to.equal(false);
-    expect(wrapper.find('select').length).to.equal(2);
     expect(wrapper.find('AnnouncementsEditor').length).to.equal(1);
-    expect(wrapper.find('CollapsibleEditorSection').length).to.equal(11);
```

Comment:
this is mostly for my understanding but why is checking for the number of StandardsEditors something we want to do when checking for CollapsibleEditorSections something we don't want to do?

Reply:
Fair question :-) I can't say I have the most well thought out answer, but I was trying to find a compromise between removing the parts of this test that seem brittle and keeping the parts that help make sure we don't have a regression. The CollapsibleEditorSection seems like something we'd need to update every time we add a section, while the checks for individual editor types seem like they help protect against regressions (a specific editor disappearing) without being as likely to break when more editor sections are added or removed.

You could argue that updating the CollapsibleEditorSection count shouldn't be that bad because if you are making a big change (adding/removing a section) then you should be running the tests in this file anyway and probably adding/updating a test.

I think we haven't really gotten to a point where we have a great plan for how we like to write unit tests, at least I don't have one in my head. we have some opposition to brittle tests but not a great idea of what kind of tests we do want in their place.

Reply:
Ya recently I've been feeling like unit tests are a moving target and I'm not sure how to write them anymore. I would love to get more clarity here. To me it feels like CollapsibleEditorSection only feels brittle because we are making so many changes so quickly but it actually makes sure this very large page with lots of sections is loading right.